modular_citadel/code/game/gamemodes/gangs/gang_datums.dm

```diff
@@ -0,0 +1,139 @@
+// Those are all required. If one is missed, stuff could break.
+
+/datum/team/gang/clandestine
+	name = "Clandestine"
+	color = "#FF0000"
+	inner_outfits = list(/obj/item/clothing/under/syndicate/combat)
+	outer_outfits = list(/obj/item/clothing/suit/jacket)
+
+/datum/team/gang/prima
+	name = "Prima"
+	color = "#FFFF00"
+	inner_outfits = list(/obj/item/clothing/under/color/yellow)
+	outer_outfits = list(/obj/item/clothing/suit/hastur)
+
+/datum/team/gang/zerog
+	name = "Zero-G"
+	color = "#C0C0C0"
+	inner_outfits = list(/obj/item/clothing/under/suit_jacket/white)
+	outer_outfits = list(/obj/item/clothing/suit/hooded/wintercoat)
+
+/datum/team/gang/max
+	name = "Max"
+	color = "#800000"
+	inner_outfits = list(/obj/item/clothing/under/color/maroon)
+	outer_outfits = list(/obj/item/clothing/suit/poncho/red)
+
+/datum/team/gang/blasto
+	name = "Blasto"
+	color = "#000080"
+	inner_outfits = list(/obj/item/clothing/under/suit_jacket/navy)
+	outer_outfits = list(/obj/item/clothing/suit/jacket/miljacket)
+
+/datum/team/gang/waffle
+	name = "Waffle"
+	color = "#808000" //shared color with cyber, but they can keep brown cause waffles.
+	inner_outfits = list(/obj/item/clothing/under/suit_jacket/green)
+	outer_outfits = list(/obj/item/clothing/suit/poncho)
+
+/datum/team/gang/north
+	name = "North"
+	color = "#00FF00"
+	inner_outfits = list(/obj/item/clothing/under/color/green)
+	outer_outfits = list(/obj/item/clothing/suit/poncho/green)
+
+/datum/team/gang/omni
+	name = "Omni"
+	color = "#008080"
+	inner_outfits = list(/obj/item/clothing/under/color/teal)
+	outer_outfits = list(/obj/item/clothing/suit/studentuni)
+
+/datum/team/gang/newton
+	name = "Newton"
+	color = "#A52A2A"
+	inner_outfits = list(/obj/item/clothing/under/color/brown)
+	outer_outfits = list(/obj/item/clothing/suit/toggle/owlwings)
+
+/datum/team/gang/cyber
+	name = "Cyber"
+	color = "#00f904" //Cyber and waffle shared colors, I made these guys green and made weed darker green.
+	inner_outfits = list(/obj/item/clothing/under/color/lightbrown)
+	outer_outfits = list(/obj/item/clothing/suit/nemes)
+
+/datum/team/gang/donk
+	name = "Donk"
+	color = "#0000FF"
+	inner_outfits = list(/obj/item/clothing/under/color/darkblue)
+	outer_outfits = list(/obj/item/clothing/suit/apron/overalls)
+
+/datum/team/gang/gene
+	name = "Gene"
+	color = "#00FFFF"
+	inner_outfits = list(/obj/item/clothing/under/color/blue)
+	outer_outfits = list(/obj/item/clothing/suit/apron)
+
+/datum/team/gang/gib
+	name = "Gib"
+	color = "#636060" //Applying black to grayscale... Zero-G is already grey too. oh well.
+	inner_outfits = list(/obj/item/clothing/under/color/black)
+	outer_outfits = list(/obj/item/clothing/suit/jacket/leather/overcoat)
+
+/datum/team/gang/tunnel
+	name = "Tunnel"
+	color = "#FF00FF"
+	inner_outfits = list(/obj/item/clothing/under/villain)
+	outer_outfits = list(/obj/item/clothing/suit/poncho/ponchoshame)
```

Comment:
Isnt ponchoshame no drop?

Reply:
Its cursed, yep. Got anything to replace it with?

Reply:
Swapped the Poncho of shame out with a leather jacket since tunnel gang, changed some clothing around.